src/lexer.h

```diff
@@ -115,6 +124,20 @@ uint32_t is () {
 uint32_t ie () {
   return import_read_head->end - source;
 }
+// getImportStatementStart
+uint32_t iss () {
+  if (import_read_head->dynamic == STANDARD_IMPORT)
+    return import_read_head->statement_start - source;
+  else
+    return -1;
```

Comment:
So I think we can just remove this path and the doubling up should work fine?

Reply:
sgtm, done